Error: Vehicle type of vehicle 'ToC_veh' ('anothertype') must coincide with manualType ('manual') or automatedType ('automated') specified for its ToC-device (or drawn from the specified vTypeDistributions).
Quitting (on error).
Traceback (most recent call last):
  File "/home/leo/repos/sumo_master/sumo/tests/../tests/complex/sumo/ToCDevice/invalid_vType/runner.py", line 97, in <module>
    traci.start([sumoBinary, "-n", "input_net.net.xml", "-r", "input_routes.rou.xml", "--no-step-log", "true"])
  File "/home/leo/repos/sumo_master/sumo/tools/traci/__init__.py", line 88, in start
    return getVersion()
  File "/home/leo/repos/sumo_master/sumo/tools/traci/__init__.py", line 160, in getVersion
    return _connections[""].getVersion()
  File "/home/leo/repos/sumo_master/sumo/tools/traci/connection.py", line 290, in getVersion
    result = self._sendExact()
  File "/home/leo/repos/sumo_master/sumo/tools/traci/connection.py", line 98, in _sendExact
    raise FatalTraCIError("connection closed by SUMO")
traci.exceptions.FatalTraCIError: connection closed by SUMO
